src/autoscaler/api/policyvalidator/policy_validator_test.go

```diff
@@ -375,8 +381,8 @@ var _ = Describe("PolicyValidator", func() {
 				It("should fail", func() {
 					Expect(errResult).To(Equal([]PolicyValidationErrors{
 						{
-							Context:     "(root)",
-							Description: "json: cannot unmarshal number 90.55 into Go struct field ScalingRule.scaling_rules.threshold of type int64",
+							Context:     "(root).scaling_rules.0.threshold",
+							Description: "Invalid type. Expected: integer, given: number",
```

Comment:
The current definition for that validation directly checks first against the schema before loading everything into golang-values. Hence we get the intentional json-validation-error-structure here.